pyproject.toml

```diff
@@ -148,7 +151,6 @@ pydoc-markdown = "^4.1.6"
 pytest-timeout = "^1.4.2"
 mypy = "^0.910"
 bandit = "^1.6.3"
-typing-extensions = "^3.7.4"
```

Comment:
Why do we need to move it from dev dependencies?

Reply:
it's now a prod dependency as we need it for our dataclasses